README.md

```diff
@@ -45,6 +45,9 @@ current working directories `node_modules` folder, then this version of eslint
 is going to be used. Otherwise, the version of eslint that ships with
 `eslint_d` is used as a fallback.
 
+It's possible to force `eslint_d` to only resolve local `eslint` by setting the
+`ESLINT_D_LOCAL_ONLY` environment variable to a truthy value (ie. `true` or `1`).
```

Comment:
This is not entirely correct the way it's implemented. The environment variable is a string, so `"0"` or `"false"` would also be truthy 😉

Reply:
Yeah, I know - the values `true` and `1` was more examples. I'll change the docs to be more clear though.